src/quantuminspire/qiskit/quantum_inspire_provider.py

```diff
@@ -60,6 +76,7 @@ def set_authentication_details(self, email: str, password: str, qi_url: str = QI
             qi_url: URL that points to quantum-inspire api. Default value: 'https://api.quantum-inspire.com'.
 
         """
+        del qi_url
```

Comment:
why do you delete the parameter instead of passing it along to `set_basic_authentication` ?

Reply:
I resolved an unused parameter warning without changing the impact, but I think you are right. This should be in the call to 'set_basic_authentication'